jax/test_util.py

```diff
@@ -695,6 +702,15 @@ def cases_from_gens(*gens):
       yield ('_{}_{}'.format(size, i),) + tuple(gen(size) for gen in gens)
 
 
+class JaxTestLoader(absltest.TestLoader):
+  def getTestCaseNames(self, testCaseClass):
+    names = super().getTestCaseNames(testCaseClass)
```

Comment:
Do these include the class name, or just the test method name?

Reply:
These are just the test nethod names.

Reply:
Maybe do something like `names = [testCaseClass.__name__ + name for name in names]`? When I saw this, I assumed I could use the class name as a regexp e.g. to run an entire test class (so `--test_targets` is strictly more expressive than putting a test name or class on the command line). I'm very anchored on the C++ test runner btw :)

Reply:
You read my mind!!

Reply:
I changed it so that the regex is over the full name; I think that will be useful.